### DISCUSSION TITLE
Cloud border adjustment?

### DISCUSSION THREAD

**Discussion by autoca** [freeplane/freeplane#622] [Help] [answered]
Hi all,
I am not sure and have not found anything, but before I add a feature request, I thought I better ask first:

Can the border of a cloud be adjusted?
Specifically in color and thickness, but maybe also in style (dotted, ...).

Regards, autoca

**Comment by macmarrum** [comment, marked as answer]
No, border of a cloud cannot be adjusted.

**Comment by autoca** [comment]
ok, thanks
